docs/src/main/tut/docs/config-cursors.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: docs
+title: Config Cursors
+---
+
+## {{page.title}}
+
+When a `ConfigReader` needs to be created from scratch, users need to implement a `from` method with the following
+signature:
+
+```scala
+def from(cur: ConfigCursor): Either[ConfigReaderFailures, A]
+```
+
+The `ConfigCursor` class is a wrapper for the raw `ConfigValue` provided by Typesafe Config. It provides an idiomatic,
```

Comment:
Are we unable to link to the `ConfigCursor` docs because it hasn't appeared in a release yet?

Reply:
That's right. Since our CI build is checking for broken links on these pages, I can't even put it in advance for the release of version 0.9.0 😄